java-squid/src/main/java/org/sonar/java/se/ConstraintManager.java

```diff
@@ -130,21 +133,22 @@ private static Tree skipTrivial(Tree expression) {
 
   }
 
-  public boolean isNull(ProgramState ps, SymbolicValue val){
+  public boolean isNull(ProgramState ps, SymbolicValue val) {
     return NullConstraint.NULL.equals(ps.constraints.get(val));
   }
 
   public Pair<ProgramState, ProgramState> assumeDual(ProgramState programState, Tree condition) {
-    //FIXME condition value should be evaluated to determine if it is worth exploring this branch. This should probably be done in a dedicated checker.
+    // FIXME condition value should be evaluated to determine if it is worth exploring this branch. This should probably be done in a
+    // dedicated checker.
     condition = skipTrivial(condition);
     switch (condition.kind()) {
       case INSTANCE_OF: {
         InstanceOfTree instanceOfTree = (InstanceOfTree) condition;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 11. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)